Hearthstone Deck Tracker/FlyoutControls/Options/Tracker/TrackerSettings.xaml.cs

```diff
@@ -357,5 +359,33 @@ private void CheckBoxAnalytics_OnUnchecked(object sender, RoutedEventArgs e)
 			Config.Instance.GoogleAnalytics = false;
 			Config.Save();
 		}
-	}
+
+        private void CheckboxShowNewsBar_Checked(object sender, RoutedEventArgs e)
+        {
+            if (!_initialized)
+                return;
+            Config.Instance.IgnoreNewsId = -1;
+            Config.Instance.ShowNewsBar = true;
+            Config.Save();
+            Utility.NewsUpdater.UpdateAsync();
+        }
+
+        private void CheckboxShowNewsBar_Unchecked(object sender, RoutedEventArgs e)
+        {
```

Comment:
Extracting `MainWindow.BtnCloseNews_OnClick()` to a new method and calling it here would be much cleaner.